tests/bdd/zfs.feature

```diff
@@ -253,3 +253,24 @@ Feature: Validate volume provisioning for fsType zfs, ext4, xfs, btrfs
       |  ext4  |
       |  xfs   |
       |  btrfs |
+
+  Scenario Outline: Main features should work after upgrade
+    Given The latest stable version of OpenEBS Local PV ZFS is installed in the cluster
+    And a storage class is created with fsType as <fsType> and thinprovision as <thinprovision> and allowexpansion as true
+    And pvc is created referencing this storage class and a deployment using the same pvc
```

Comment:
please add snapshot creation steps so that after upgrade restore is created from snapshot